dev/TeachingTip/TeachingTip.xaml

```diff
@@ -498,7 +500,9 @@
                                       Grid.Row="1"
                                       Grid.Column="1"
                                       Grid.ColumnSpan="3"
-                                      Grid.RowSpan="3">
+                                      Grid.RowSpan="3"
+                                      contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
+                                      contract7NotPresent:CornerRadius="{StaticResource OverlayCornerRadius}">
```

Comment:
Does this look correct? I think it probably belongs on the tail occlusion grid, or perhaps its parent.

Reply:
Setting on parent didn't work, I think it's because of z orders, the inner one is higher and covering the parents.